### PR TITLE
register egjs

### PR DESCRIPTION
**Pull request by sculove**
I'd like to add egjs to jsdelivr CDN

### CODE REVIEW

File: files/egjs/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "github",
+  "name": "egjs",
+  "repo": "naver/egjs",
+  "files": {
+    "basePath": "dist/",
+    "include": ["**/*.js"]
```
Comment (jimaek) [inline files/egjs/update.json]: You can remove this line, by default all files in that folder will be added

File: files/egjs/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author = "NAVER corp"
+github = "https://github.com/naver/egjs"
+homepage = "http://naver.github.io/egjs"
+description = "Set of UI interactions, effects and utilities components library."
+mainfile = "dist/eg.js"
```
Comment (jimaek) [inline files/egjs/info.ini]: Just `eg.min.js`

File: files/egjs/update.json (hunk 1 above)
Comment (sculove) [inline files/egjs/update.json]: Thank you

File: files/egjs/info.ini (hunk 2 above)
Comment (sculove) [inline files/egjs/info.ini]: I don't understand your intention
Our main files are eg.js and eg.min.js
What should I do?

Comment (jimaek) [inline files/egjs/info.ini]: You should use `eg.min.js` as the parameter for mainfile

**Comment by sculove** [comment]
is anyone progressing?

File: files/egjs/info.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+author = "NAVER corp"
+github = "https://github.com/naver/egjs"
+homepage = "http://naver.github.io/egjs"
+description = "Set of UI interactions, effects and utilities components library."
+mainfile = "dist/eg.min.js"
```
Comment (jimaek) [inline files/egjs/info.ini]: This line is still incorrect. It should be just `eg.min.js`

Comment (sculove) [inline files/egjs/info.ini]: @jimaek OK. thank you

Comment (sculove) [inline files/egjs/info.ini]: @jimaek I applied. please, update

**Pull request merged**